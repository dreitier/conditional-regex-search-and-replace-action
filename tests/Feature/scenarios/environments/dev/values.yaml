first: value
second: value2
  imageTag: "0.0.666" old_custom_var
fourth: value4
  # the next line gets not replaced as the regex does not match the line
  imageTag: "0.0.666"