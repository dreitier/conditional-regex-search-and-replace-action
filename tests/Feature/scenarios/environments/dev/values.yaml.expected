first: value
second: value2
  imageTag: "1.0.0" 4.0.0
fourth: value4
  # the next line gets not replaced as the regex does not match the line
  imageTag: "0.0.666"